aws/cloudformation-templates/segment.yaml

```diff
@@ -51,6 +57,10 @@ Resources:
         S3Key: !Sub '${ResourceBucketRelativePath}aws-lambda/segment-personalize-inference-destination.zip'
       Runtime: python3.8
       Timeout: 900
+      VpcConfig:
+        SecurityGroupIds:
+          - !Ref LambdaVpcSecurityGroup
+        SubnetIds: !Split [",", !Ref LambdaVpcSubnets]
```

Comment:
Since the [Segment Personalize destination](https://segment.com/docs/connections/destinations/catalog/amazon-personalize/) calls this function from Segment's account, we should make sure that this call will still succeed with the function in a VPC.

Reply:
Good shout.  We can verify by performing a cross account call. 
However, I believe it should be fine since access to the Lambda is controlled by IAM - whereas the changes that have been made to the configuration are to allow the Lambda access to the VPC, which it achieves by deploying an ENI into the subnet.